srcpkgs/openpgp-card-tools/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'openpgp-card-tools'
+pkgname=openpgp-card-tools
+version=0.0.6
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config llvm clang"
+makedepends="nettle-devel pcsclite-devel"
+short_desc="Tools for inspecting, configuring and using OpenPGP cards"
+maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license="MIT, Apache-2.0"
+homepage="https://crates.io/crates/openpgp-card-tools"
+distfiles="https://static.crates.io/crates/${pkgname}/${pkgname}-${version}.crate"
+checksum=ccdd462f47fc92c0e09de78fb0a2ef5be818e836ab7c27b1fdbd54c812008d3d
+
+post_install() {
+	# Upstream doesn't include those files in the crate tarballs,
+	# and doesn't have git tags right now
```

Comment:
has a request to upstream for this been made?

Reply:
Nope, not yet. One of the reasons this PR is still marked as a draft

Reply:
Updated the `TODO` section in the PR description, to include this. Resolving thread.